sources/Valkey.Glide/Errors.cs

```diff
@@ -28,6 +28,19 @@ public RequestException(string message) : base(message) { }
         public RequestException(string message, Exception innerException) : base(message, innerException) { }
     }
 
+    /// <summary>
+    /// An error returned by the Valkey server during script or function execution.
+    /// /// This includes Lua comtion errors, runtime errors, and script/function management errors.
```

Comment:
*Compilation (I assume?) and extra `///`.

```suggestion
    /// This includes Lua compilation errors, runtime errors, and script/function management errors.
```